tests/fuzzy/CMakeLists.txt

```diff
@@ -34,6 +68,6 @@ macro(_add_test test)
 endmacro()
 
 _add_test(key_provider_test_fuzzy)
```

Comment:
Change test names to format `fuzzy_test__<module>__<function>` for all test source files.
For instance, `fuzzy_test__key_asn1_deserializer__import_public_key.c`.